src/main/java/com/google/cloud/spanner/pgadapter/wireprotocol/QueryMessage.java

```diff
@@ -31,11 +32,11 @@
 
   public QueryMessage(ConnectionHandler connection) throws Exception {
     super(connection);
+    String query = StatementParser.removeCommentsAndTrim(this.readAll());
```

Comment:
Wouldn't this result in all queries we sent to the backend lack comments? I'd hope that we keep comments when it's possible to do so. Thoughts?

Reply:
The comments will be stripped out anyways. I think this is ok.